addOns/pscanrulesBeta/src/main/java/org/zaproxy/zap/extension/pscanrulesBeta/StrictTransportSecurityScanRule.java

```diff
@@ -101,7 +101,7 @@ private void raiseAlert(VulnType currentVT, String evidence, HttpMessage msg, in
                 .setSolution(getAlertElement(currentVT, "soln"))
                 .setReference(getAlertElement(currentVT, "refs"))
                 .setEvidence(evidence)
-                .setCweId(16) // CWE-16: Configuration
+                .setCweId(693) // CWE-693: Protection Mechanism Failure
```

Comment:
Can this also use CWE-319: Cleartext Transmission of Sensitive Information, as done for Insecure Form Load / Post?